.github/workflows/schedule.py

```diff
@@ -167,21 +174,38 @@ def create_issue(self, title_date: date) -> None:
         if issues:
             log.info('At least one matching issue already exists: %r', issues)
         else:
-            assignees = self.properties.get('assignees')
-            if assignees is not None:
-                flags.append(f'--assignee={assignees}')
             command = [
                 'gh', 'issue', 'create',
                 *flags,
                 f'--title={title}',
-                f'--label={self.properties["labels"]}',
                 f'--body={self.body}'
             ]
+            edit_command, flags = ['gh', 'issue', 'edit', *flags], []
+
+            def accumulate_edit_command_flags(option, target):
+                flag = f'--add-{option}={target}'
+                if target:
+                    flags.append(flag)
+                else:
+                    require(target is None, requirement_message, f'{option}s')
+
+            accumulate_edit_command_flags('assignee', self.properties.get('assignees'))
+            accumulate_edit_command_flags('label', self.properties.get('labels'))
+
             if self.dry_run:
                 log.info('Would run %r', command)
+                for cmd in [edit_command + ['#ISSUE', f] for f in flags]:
+                    log.info('Followed by %r', cmd)
```

Comment:
This may not be representative of what's actually done during a wet run. If this is not clear please request a PL slot.

Reply:
Will do.